packages/app/src/cli/services/logs.ts

```diff
@@ -91,17 +92,34 @@ async function prepareForLogs(commandOptions: LogsOptions): Promise<{
     ...commandOptions,
     userProvidedConfigName: commandOptions.configName,
   })
-  let developerPlatformClient = selectDeveloperPlatformClient({configuration})
-  const devContextOptions: DevContextOptions = {...commandOptions, developerPlatformClient}
-  const {storeId, remoteApp, localApp} = await ensureDevContext(devContextOptions)
+  const developerPlatformClient = selectDeveloperPlatformClient({configuration})
+  const primaryStoreFqdn = commandOptions.storeFqdns?.[0]
+  const devContextOptions: DevContextOptions = {
+    ...commandOptions,
+    storeFqdn: primaryStoreFqdn,
+    developerPlatformClient,
+  }
+  const {storeId, storeFqdn, remoteApp, localApp} = await ensureDevContext(devContextOptions)
 
-  developerPlatformClient = remoteApp.developerPlatformClient ?? developerPlatformClient
```

Comment:
I inlined this action into the return value below- this allows me to use `developerPlatformClient` in a `Promise.all` block, since it's now constant.